Decred Wallet/Custom Views/FloatingLabelTextInput.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  FloatingTextInput.swift
+//  Decred Wallet
+//
+//  Created by Sprinthub on 03/09/2019.
+//  Copyright © 2019 Decred. All rights reserved.
+//
+
+import UIKit
+
+class FloatingLabelTextInput: UITextField {
```

Comment:
I'm not sure why we need 2 subclasses of `UITextField`. Seems the other one (`FloatingLabelTextField`) is not used. I prefer that name though. See if we can keep and use that file instead. Remember to apply review comments in that file here as well if you're keeping this file.

Reply:
don't quite understand. `FloatingLabelTextInput` only subclasess `UITextField` once

Reply:
i renamed `floatingLabelTextField` to `floatingLabelTextInput` cuz i felt input sounded more self explanatory